windows/source/bin/Release/serviceConfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Bahmni>
+  <serviceSettings>
+    <VM_NAME>Bahmni</VM_NAME>
+    <LOGS_PATH>C:\Bahmni\Bahmni Service Logs</LOGS_PATH>

```

Comment:
Can you please update this path to C:\EMR\bahmni\bahmni-service-logs?

Do we need to create this folder manually?